src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Configuration/ApplicationInsightsSinkExceptionOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using GuardNet;
+
+namespace Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights.Configuration
+{
+    /// <summary>
+    /// User-defined configuration options to influence the behavior of the Azure Application Insights Serilog sink while tracking exceptions.
+    /// </summary>
+    public class ApplicationInsightsSinkExceptionOptions
+    {
+        private static readonly Regex PropertyFormatRegex = new Regex(@"[\{\{0\}\}]", RegexOptions.Compiled);
+        private string _propertyFormat = "Exception-{0}";
+
+        /// <summary>
+        /// Gets or sets the flag indicating whether or not the properties of the exception should be included as custom dimensions in the exception tracking.
+        /// </summary>
+        /// <remarks>
+        ///     Only the current-level properties are considered during the exception tracking; inherited properties will be excluded.
+        /// </remarks>
+        public bool IncludeProperties { get; set; } = false;
```

Comment:
Can you document this flag please?

Reply:
I (already) updated the docs with this property, is that what you meant?